R/bootstrap.nlsfit.R

```diff
@@ -242,6 +248,9 @@ bootstrap.nlsfit <- function(fn,
                              y,
                              x,
                              bsamples,
+                             priors = list(param = c(), p = c(), psamples = c()),
+                             msamples = c(),
```

Comment:
As explained below, I think that `msamples` and `np` does not belong into this function.